dotnet/sample/AutoGen.BasicSamples/Example16_Anthropic.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Example16_Anthropic.cs
+
+using AutoGen.Anthropic;
+using AutoGen.Anthropic.Middleware;
+using AutoGen.Anthropic.Utils;
+using AutoGen.Core;
+
+namespace AutoGen.BasicSample;
```

Comment:
I'm going to create specific sample project for each package (e.g. `AutoGen.OpenAI.Samples`, `AutoGen.SemanticKernel.Samples`)...  Could you maybe move this sample to `AutoGen.Anthropic.Samples` as well